packages/ses/src/lockdown-shim.js

```diff
@@ -115,6 +117,17 @@ export function repairIntrinsics(
 
   const intrinsics = intrinsicsCollector.finalIntrinsics();
 
+  // Wrap console unless suppressed.
+  // At the moment, the console is considered a host power in the start
+  // compartment, and not a primordial. Hence it is absent from the whilelist
+  // and bypasses the intrinsicsCollector.
```

Comment:
When we have a more complete story for vetted shims, does expressly threading and taming with our specific causal console dependency become less desirable? I imagine people will want to bring their own causal console implementation to the table. Is the expectation that they would implement a shim that replaces the global console before lockdown, which in turn gets wrapped by our causal console?

Another question: is our causal console coupled closely enough to `ses-shim` that no other package is likely to depend upon it?

Reply:
Let's discuss.